code/datums/uplink_item.dm

```diff
@@ -278,7 +276,9 @@ var/list/uplink_items = list()
 	reference = "PG"
 	item = /obj/item/clothing/gloves/color/yellow/power
 	cost = 10
-	job = list("Station Engineer","Chief Engineer")
+	job = list(
```

Comment:
If you're going to start pulling the same thing that TG coder did for line-breaking lists and other text, I'm going to tell you the same thing they effectively did: we'll hunt you down and shank you in your sleep.

Stated otherwise, please don't do this.